src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,31 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum Rank {
+    FIRST(new LottoMatch(6, false), 2_000_000_000),
```

Comment:
사실상 3개의 필드를 가지는 상수같은데 아래와 같은 방식은 어떨까요~?
```suggestion
    FIRST(6, false, 2_000_000_000),
```